exercises/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+func insertSort(x []int) {
+	for i := 1; i < len(x); i++ {
+		for j := i; j > 0; j-- {
+			if x[j-1] > x[j] { // <-- on this sign depends if slice will be sorted in ascending or descending order
+				x[j-1], x[j] = x[j], x[j-1]
+			}
+		}
+	}
+}
+
+//Found this quite interesting piece on https://medium.com/star-gazers/insertion-sorting-algorithm-in-go-26f1ec49936c
+//It was written on base of pseudocode
+// But helped to understand
+// No if statements? so must a lot loop iterations
+func insertSortBySteveHook(x []int) []int {
```

Comment:
This one is good! It is better than the previous one. Though the insertion sort is generally very slow